packages/build/bin/build-css.js

```diff
@@ -28,8 +27,8 @@ program
   .option('-g, --useGlamor', 'Compile css using glamor', false)
   .parse(process.argv)
 
-const inputFilePath = program.input
-const outputDirectoryPath = program.output
+const inputFilePath = path.join(process.cwd(), program.input)
```

Comment:
Was this required to get the input cli flag to work?

Reply:
Yep